docs/security/trust/trust_sandbox.md

```diff
@@ -101,8 +101,6 @@ So, you'll need an entry for both the servers in your local `/etc/hosts` file.
         RUN chmod +x /usr/bin/docker \
           && apt-get update \
           && apt-get install -y \
-          tree \
```

Comment:
vim is useful to tamper data and tree is super useful to list all the keys that are generated.